### DISCUSSION TITLE
How to change FreshRSS logo ? | Comment changer le logo FreshRSS ?

### DISCUSSION THREAD

**Discussion by Marc-Harony** [FreshRSS/FreshRSS#4734] [Q&A] [answered]
Hi 👋🏻 , I'm wandering if it is possible to change the FreshRSS logo on the top left and the tab icon, and if yes, how to do that ?
I've tried to edit the config.default and config-user files but it seems to be inefficient.

I've also tried to replace the file in `p/themes/icons/FreshRSS-logo.svg` by the custom one but it has been removed by the update.
Thanks for your help !

---

Bonjour👋🏻, je me demandais s'il était possible de changer le logo FreshRSS en haut à gauche du site ainsi que l'icône sur l'onglet du navigateur et si oui, comment le faire ? J'ai essayé de modifier les fichiers config.default et config-user mais sans succès.

J'ai aussi essayé de remplacer le fichier logo présent dans `p/themes/icons/FreshRSS-logo.svg`. Ça a fonctionné au début mais il a été remplacé par l'original après une mise-à-jour.
Merci pour votre aide !

**Comment by Alkarex** [comment, marked as answer]
See the `logo_html` section of your `./data/config.php`

```php
'logo_html' => '<img class="logo" src="http://example.net/logo.png" alt="Custom Logo" />',
```

https://freshrss.github.io/FreshRSS/en/admins/03_Installation.html

Ref https://github.com/FreshRSS/FreshRSS/pull/4369

**Reply by Marc-Harony** [reply]
Oh thanks, that worked ! 🙏🏻 
Do you also know how to change this icon ? 
![image](https://user-images.githubusercontent.com/88329570/195868630-c4df9311-8e54-43e3-a1f2-3f631b8f14a6.png)

**Reply by Marc-Harony** [reply]
I realize that this screenshot isn't really clear.
I'm talking about the icon on the borwser's tab : 
![image](https://user-images.githubusercontent.com/88329570/195870717-7063499d-4b2c-4ecd-927e-41e5b5c86967.png)

**Reply by Alkarex** [reply]
There is no option to customise the favicon at the moment

**Reply by Marc-Harony** [reply]
Ok, noted.
Thanks a lot for your help !

**Reply by KatyPurry** [reply]
Is it possible not to use favicons that say 1k, 2k, etc?

**Reply by Alkarex** [reply]
@KatyPurry Please open a new question for that 🤓
